docs/pages/connect-your-client/tsh.mdx

```diff
@@ -80,10 +80,12 @@ to call [`tsh login`](../reference/cli.mdx#tsh-login) in the beginning.
 
 ## Installing tsh
 
-Follow [these install instructions](../installation.mdx) to obtain the `tsh`
-binary. Ideally, install `tsh` of the same version as the version used in your
+Follow install instructions to obtain the `tsh` binary. Install
```

Comment:
Is there an easy way for end users to determine the major version of the Teleport cluster (e.g., `/webapi/ping`)? 

The instructions below also don't allow the user to select a version. Maybe turn the version within `install-tsh.mdx` into a parameter and, in this inclusion, use a `Var` component for the value of the parameter?

Reply:
`tsh version` will show the version of the connected proxy, as long as `--client-only` isn't specified.

Reply:
WE can add a expandable option to get their current version

Reply:
Please see update @ptgott